app/presenters/provider/admin/backend_apis_index_presenter.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class Provider::Admin::BackendApisIndexPresenter
+  include System::UrlHelpers.system_url_helpers
+
+  def initialize(current_account:, params: {})
+    @current_account = current_account
+    @pagination_params = { page: params[:page] || 1, per_page: params[:per_page] || 20 }
+    @search = ThreeScale::Search.new(params[:search] || params)
```

Comment:
This seems a little bit vague. What kind of data we allow in API documentation? Can we be a little be more strict in setting search params?

Reply:
If `:search` param is not passed, then whole `params` is being used. Shouldn't use provide `:search` param in call? Why fallback? And if we fall back, shouldn't we at least remove other known parameters like `:page` and `:per_page`.

I don't know how things work so excuse me if what I ask is obvious already.